languages/csharp/exercises/concept/exceptions/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+static class SimpleCalculator
+{
+    public static string ErrorLog {get; set;}
+    public static int Calculate(int operand1, int operand2, string operation)
+    {
+        int result = 0;
+        ErrorLog = "";
+        try
+        {
+            switch(operation)
+            {
+                case "+":
+                    result = Addition(operand1, operand2);
+                    break;
+                case "*":
+                    result = Multiplication(operand1, operand2);
+                    break;
+                case "/":
+                    result = Division(operand1, operand2);
+                    break;
+                default:
+                    throw new InvalidOperationException($"Operation {operation} does not exist");
+            }
+        }
+        catch(ArgumentException e)
+        {
+            ErrorLog = $"Result invalid: {e.Message}";
+            return -1;
+        }
+        return result;
+
+    }
+
+    private static int Division(int operand1, int operand2)
+    {
+        return operand1 / operand2;
+    }
+
+    private static int Multiplication(int operand1, int operand2)
+    {
+        int result = operand1 * operand2;
+        if(operand1 > 0 && operand2 > 0 && result < 0)
+        {
+            throw new ArgumentException("Result of operation does not fit in type of int.");
+        }
+        return result;
```

Comment:
If this exercise has `overflows` as a prerequisite concept, we can have the student use `checked` here. We could then have the student catch `OverflowException` and do something with them. Possibly throwing an `ArgumentException` instead.

```suggestion
        checked
        {
            return operand1 * operand2;
        }
```

Reply:
I've implemented this suggestion and the following in the Example.cs on my local branch and will push as soon as test are ready.

Reply:
Sweet! Let me know when you want me to a new review.